linux_os/guide/services/ntp/chronyd_specify_remote_server/rule.yml

```diff
@@ -0,0 +1,35 @@
+documentation_complete: true
+
+prodtype: fedora,ocp4,ol8,rhel8
+
+title: 'A remote NTP server for Chrony is configured'
+
+description: |-
+    chrony is a daemon which implements the Network Time Protocol (NTP) is designed to
```

Comment:
```suggestion
    Chrony is a daemon which implements the Network Time Protocol (NTP). Chrony is designed to
```

Reply:
@vojtapolasek You missed capitalization of `Chrony` here.